rules/windows/file/file_access/file_access_win_susp_process_access_browser_cred_files.yml

```diff
@@ -0,0 +1,195 @@
+            - '\Chromium.exe'
+            - '\GhostBrowser.exe'
+            - '\CentBrowser.exe'
+            - '\Xvast.exe'
+            - '\Chedot.exe'
+            - '\SuperBird.exe'
+            - '\360Browser.exe'
+            - '\360Chrome.exe'
+            - '\dragon.exe'
+            - '\brave.exe'
+            - '\torch.exe'
+            - '\UCBrowser.exe'
+            - '\BliskBrowser.exe'
+            - '\Epic Privacy Browser.exe'
+            - '\nichrome.exe'
+            - '\AmigoBrowser.exe'
+            - '\KometaBrowser.exe'
+            - '\XpomBrowser.exe'
+            - '\msedge.exe'
+            - '\LiebaoBrowser.exe'
+            - '\AvastBrowser.exe'
+            - '\Kinza.exe'
+            - '\seamonkey.exe'
+            - '\icedragon.exe'
+            - '\cyberfox.exe'
+            - '\SlimBrowser.exe'
+            - '\palemoon.exe'
+    filter_main_path:
+        Image|contains:
+            - '\Sputnik\'
+            - '\MapleStudio\'
+            - '\QIP Surf\'
+            - '\BlackHawk\'
+            - '\7Star\'
+            - '\Fenrir Inc\'
+            - '\CatalinaGroup\'
+            - '\Google\'
+            - '\Coowon\'
+            - '\CocCoc\'
+            - '\uCozMedia\'
+            - '\Tencent\'
+            - '\Orbitum\'
+            - '\Slimjet\'
+            - '\Iridium\'
+            - '\Vivaldi\'
+            - '\Chromium\'
+            - '\GhostBrowser\'
+            - '\CentBrowser\'
+            - '\Xvast\'
+            - '\Chedot\'
+            - '\SuperBird\'
+            - '\360Browser\'
+            - '\360Chrome\'
+            - '\Comodo\'
+            - '\BraveSoftware\'
+            - '\Torch\'
+            - '\UCBrowser\'
+            - '\Blisk\'
+            - '\Epic Privacy Browser\'
+            - '\Nichrome\'
+            - '\Amigo\'
+            - '\Kometa\'
+            - '\Xpom\'
+            - '\Microsoft\'
+            - '\Liebao7\'
+            - '\AVAST Software\'
+            - '\Kinza\'
+            - '\Mozilla\'
+            - '\8pecxstudios\'
+            - '\FlashPeak\'
+            - '\Moonchild Productions\'
+    filter_main_legit_programs:
+        Image|startswith:
+            - 'C:\Program Files\'
+            - 'C:\Program Files (x86)\'
+            - 'C:\Windows\System32\'
+            - 'C:\Windows\SysWOW64\'
+    filter_optional_defender:
+        Image|contains: '\Microsoft\Windows Defender\'
+        Image|endswith:
+            - '\MpCopyAccelerator.exe'
+            - '\MsMpEng.exe'
+    filter_optional_thor:
+        Image|endswith:
+            - '\thor.exe'
+            - '\thor64.exe'
+    condition: all of selection_* and not 1 of filter_main_* and not 1 of filter_optional_*
+falsepositives:
+    - Antivirus, Anti-Spyware, Anti-Malware Software
+    - Legitimate software accessing browser data for synchronization or backup purposes.
+    - Legitimate software installed on partitions other than "C:\"
+    - Searching software such as "everything.exe"
```

Comment:
You could tune this out in the detection logic. But let's say that is optional for a `level: low` rule.